src/main/webapp/user.js

```diff
@@ -4,25 +4,22 @@ import {
 } from './js/fetch_handler.js';
 
 /**
- * Get the user @param user from the user servlet. gets a user object from the
- * UserServlet
- * @param {string} user
+ * Get the user with @param userID from the user servlet. If no @param userID is
+ * supplied, function returns the currently signed in user.
+ * @param {string=} userID OPTIONAL
  * @return {dashboard.User}
  */
-function getUser(user) { //eslint-disable-line  
-  // TODO: switch to standard fetch error handler
-  const fetchRequest = fetch(makeRelativeUrlAbsolute('/user?githubID=' + user));
+function getUser(userID = null) { //eslint-disable-line  
```

Comment:
nit: what was the eslint error on this line? Let's try to be specific. If it no longer applies, we can remove the exception.

Reply:
I'm not sure, I'll check
```suggestion
function getUser(userID = null) {
```

Reply:
It doesn't show up anymore. I'll make a PR that looks into making the disable-line comments more specific.